releasenotes/notes/avoid-data-race-otlp-90e19c52940c4eaa.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
```

Comment:
I think this needs to be in 7.48, so may not need this release note

Reply:
Discussed offline: since this is not a regression it will not make it into 7.48 (at least for the time being)